src/audio/kpb.c

```diff
@@ -336,26 +336,28 @@ static int kpb_prepare(struct comp_dev *dev)
 	if (ret == COMP_STATUS_STATE_ALREADY_SET)
 		return PPL_STATUS_PATH_STOP;
 
+	/* Init private data */
 	cd->kpb_no_of_clients = 0;
+	cd->buffered_data = 0;
 
-	/* init history buffer */
+	/* Init history buffer */
```

Comment:
These comment updates should be a separate patch.

Reply:
Done.